tests/agents/test_microchain.py

```diff
@@ -33,39 +34,46 @@ def before_all_tests() -> t.Generator[None, None, None]:
     yield None
 
 
-def test_get_markets() -> None:
-    get_markets = GetMarkets()
+@pytest.mark.parametrize("market_type", [MarketType.OMEN])
```

Comment:
Is it correct that they are tested only for Omen?

Reply:
Good point. For this one it should work for all markets (except isn't working) for polymarket atm. Will investigate separately.

The others are currently Omen specific, but once we have a 1-1 mapping for a AgentMarket.some_function -> Function we can do `@pytest.mark.parametrize("market_type", list(MarketType))`